CHANGES.md

```diff
@@ -1,5 +1,187 @@
+    extensions namespaces metadata (so they are visible by `linkerd check`),
+    replaced by the new `extension-init` image
+  * Converted `ServerAuthorization` resources to `AuthorizationPolicy` resources
+    in Linkerd extensions
+  * Removed policy resources bound to admin servers in extensions (previously
+    these resources were used to authorize probes but now are authorized by
+    default)
+  * Fixed the link to the Jaeger dashboard the in viz dashboard (thanks
+    @eugenegoncharuk!)
+  * Updated linkerd-jaeger's collector to expose port 4318 in order support HTTP
+    alongside gRPC (thanks @uralsemih!)
+
+* Among other dependency updates, the no-longer maintained ghodss/yaml library
+  was replaced with sigs.k8s.io/yaml (thanks @Juneezee!)
+
+This release includes changes from a massive list of contributors! A special
+thank-you to everyone who helped make this release possible:
+
+Andrew Pinkham [@jambonrose](https://github.com/jambonrose)
+Arnaud Beun [@bunnybilou](https://github.com/bunnybilou)
+Carlos Tadeu Panato Junior [@cpanato](https://github.com/cpanato)
+Christian Segundo [@someone-stole-my-name](https://github.com/someone-stole-my-name)
+Dani Baeyens [@danibaeyens](https://github.com/danibaeyens)
+Duc Tran [@ductnn](https://github.com/ductnn)
+Eng Zer Jun [@Juneezee](https://github.com/Juneezee)
+Ivan Ivic [@Tyrion85](https://github.com/Tyrion85)
+Joe Bowbeer [@joebowbeer](https://github.com/joebowbeer)
+Jonathan Ogilvie [@jcogilvie](https://github.com/jcogilvie)
+Jun [@junnplus](https://github.com/junnplus)
+Loong Dai [@daixiang0](https://github.com/daixiang0)
+María Teresa Rojas [@mtrojas](https://github.com/mtrojas)
+Mo Sattler [@MoSattler](https://github.com/MoSattler)
+Oleg Vorobev [@olegy2008](https://github.com/olegy2008)
+Paul Balogh [@javaducky](https://github.com/javaducky)
+Peter Smit [@psmit](https://github.com/psmit)
+Ryan Hristovski [@ryanhristovski](https://github.com/ryanhristovski)
+Semih Ural [@uralsemih](https://github.com/uralsemih)
+Shubhodeep Mukherjee [@shubhodeep9](https://github.com/shubhodeep9)
+Siddharth S Pal [@siddharthshubhampal](https://github.com/siddharthshubhampal)
+Subhash Choudhary [@subhashchy](https://github.com/subhashchy)
+Szymon Grzemski [@sgrzemski](https://github.com/sgrzemski)
+Takumi Sue [@mikutas](https://github.com/mikutas)
+Yannick Utard [@utay](https://github.com/utay)
+Yu Cao [@yc185050](https://github.com/yc185050)
+anoxape [@anoxape](https://github.com/anoxape)
+bastienbosser [@bastienbosser](https://github.com/bastienbosser)
+bitfactory-sem-denbroeder [@bitfactory-sem-denbroeder](https://github.com/bitfactory-sem-denbroeder)
+cui fliter [@cuishuang](https://github.com/cuishuang)
+eugenegoncharuk [@eugenegoncharuk](https://github.com/eugenegoncharuk)
+h-dav @[h-dav](https://github.com/h-dav)
+martinkubrak [@martinkubra](https://github.com/martinkubra)
+verbotenj [@verbotenj](https://github.com/verbotenj)
+ziollek [@ziollek](https://github.com/ziollek)
```

Comment:
One last tweak: since this is markdown, this should be a bulleted list so it renders sanely.

Reply:
:+1: